### ISSUE TITLE
disk powered off after unlock - workaround?

### ISSUE THREAD

**Issue by x13215465** [Drive-Trust-Alliance/sedutil#477]
My poorly designed hardware (laptop by Fujistu) powers off the NVMe drive during reboot which makes use of sedutil impossible.
I can setup the drive, install PBA in shadow MBR without problem.
The system boots to PBA, password is submitted, drive unlocked, all good.
Then PBA restarts the system and unfortunately my NVMe drives loses power (as a "feature" of this particular system), get locked and it brings me back to PBA. Endless cycle...
Is there a way how to make it work in this situation? Any chance PBA can unlock the drive and boot something directly without restarting (i.e. losing power and locking the drive again)?
Thanks for your ideas

**Comment by scottcmarks** [comment]
Nope.  Same problem booting `macOS` off a TPer.  I've contacted Apple and the level of disinterest in the problem is mind-boggling.  Essentially they believe that resetting the "main" drive during boot is the prerogative of the OS and the only reliable reset is a power cycle, so ... we're sitting out of luck.
